decidim-assemblies/app/permissions/decidim/assemblies/permissions.rb

```diff
@@ -42,6 +42,8 @@ def permissions
 
         return permission_action unless assembly
 
+        view_private_users_menu?
```

Comment:
I'd prefer to use the prefix `user_can_read_` instead of view, as that's the preferred method (or at least is more used).
I'd also want to drop the `_menu` suffix, as that's only relevant to the initial bug report ("we can see the menu item"). The problem is that it isn't actually referring to the culprit of this (the whole feature of private users is available even when the space isn't private).

This is my proposal:
```suggestion
        user_can_read_private_users?
```

Feel free to improve on that :)

Reply:
fixed in [bb74aa7](https://github.com/decidim/decidim/pull/12031/commits/bb74aa7c35909f58337992441e4040338a5c8537)